samples/doc_writer_sample_output.py

```diff
@@ -276,7 +288,7 @@
                 },
                 {
                     "@type": "SupportedProperty",
-                    "property": "vocab:anotherSingleClass",
+                    "property": "https://hydrus.com/api/anotherSingleClass",
```

Comment:
It's fine to expand this here because this is for doc_writer, but make sure that we are able to change this when we parse the same document with doc_maker and we change the URI and API name in that. Not sure how you can ensure that. I would suggest not giving outputs in expanded form, and instead use expanded forms only internally in python.